packages/compartment-mapper/src/link.js

```diff
@@ -36,29 +35,6 @@ const { allSettled } = Promise;
  */
 const promiseAllSettled = allSettled.bind(Promise);
 
-const inertStaticModuleRecord = {
-  imports: [],
-  exports: [],
-  execute() {
-    throw new Error(
-      `Assertion failed: compartment graphs built for archives cannot be initialized`,
-    );
-  },
-};
-
-const inertModuleNamespace = new Compartment(
-  {},
-  {},
-  {
-    resolveHook() {
-      return '';
-    },
-    async importHook() {
-      return inertStaticModuleRecord;
-    },
-  },
-).module('');
-
```

Comment:
This is a nice deletion.

Reply:
The observable result of this deletion and using an importHook is that modules that we pass by reference no longer need to pass the brand check in SES as they're wrapped in a record. 
As a result, passing the same exit module reference to two apps/assemblies result in them receiving the module namespace that's not the same reference. 
Note it would also be the case if using policies.